feature_tests/features/product_team.failure.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Product Teams Failure Scenarios
+
+  Scenario: Failure - invalid Product Team Name
+    Given ODS Organisations:
+      | ods_code | name    |
+      | H8S7A    | BURENDO |
+    And Users:
+      | user_id          | name      |
+      | test@example.org | Test User |
+    When User "test@example.org" creates Product Team {00702d39-e65f-49f5-b9ef-6570245bfe17} called " " for supplier H8S7A
```

Comment:
few points:

1. seems like a inconsistent syntax: tabular first and then verbose second
2. it's not possible from this line to know that `test@example.org` is an ID (or is it a name or email?) and what `00702d39-e65f-49f5-b9ef-6570245bfe17` and " " correspond to.
3. Is supplier the right word? Who is the supplier of NRLF?